src/BlazorUI/Bit.BlazorUI.Extras/Styles/Cupertino/tokens.cupertino.scss

```
@@ -110,6 +110,8 @@
     --bit-layout-dialog-actions-direction: column;
     --bit-layout-dialog-actions-justify: center;
     --bit-layout-dialog-actions-align: stretch;
+    // UIAlertController is 270pt wide whatever it has to say, which is the narrowest alert of the three.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)